<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>993</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="intervaloX1">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>100</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Sans Serif</family>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="resolucion">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>240</y>
      <width>371</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Sans Serif</family>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="botonIngresar">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>530</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Sans Serif</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Ingresar</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>551</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Sans Serif</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Ingrese datos</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="etiquetaCadena">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>210</y>
      <width>330</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Sans Serif</family>
     </font>
    </property>
    <property name="text">
     <string>Resolucion</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="etiquetaER">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>330</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Sans Serif</family>
     </font>
    </property>
    <property name="text">
     <string>Intervalo x1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="resultados">
    <property name="geometry">
     <rect>
      <x>155</x>
      <y>430</y>
      <width>341</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Sans Serif</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="etiquetaER_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>330</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Sans Serif</family>
     </font>
    </property>
    <property name="text">
     <string>Intervalo y1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="poblacionInicial">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>310</y>
      <width>371</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Sans Serif</family>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="etiquetaCadena_2">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>280</y>
      <width>330</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Sans Serif</family>
     </font>
    </property>
    <property name="text">
     <string>Tam Poblacion inicial</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="etiquetaCadena_3">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>350</y>
      <width>330</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Sans Serif</family>
     </font>
    </property>
    <property name="text">
     <string>Poblacion max</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="poblacionMax">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>380</y>
      <width>371</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Sans Serif</family>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="etiquetaCadena_4">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>430</y>
      <width>330</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Sans Serif</family>
     </font>
    </property>
    <property name="text">
     <string>Probabilidad de mutacion genetica</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="pMutacionGenetica">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>460</y>
      <width>371</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Sans Serif</family>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="pMutacionIndividual">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>530</y>
      <width>371</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Sans Serif</family>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="etiquetaCadena_5">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>500</y>
      <width>330</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Sans Serif</family>
     </font>
    </property>
    <property name="text">
     <string>Probabilidad de mutacion individual</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="etiquetaER_3">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>60</y>
      <width>330</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Sans Serif</family>
     </font>
    </property>
    <property name="text">
     <string>Intervalo x2</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="intervaloX2">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>100</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Sans Serif</family>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="intervaloY1">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>170</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Sans Serif</family>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="intervaloY2">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>170</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Sans Serif</family>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="etiquetaER_4">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>140</y>
      <width>330</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Sans Serif</family>
     </font>
    </property>
    <property name="text">
     <string>Intervalo y2</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="etiquetaCadena_6">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>300</y>
      <width>330</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Sans Serif</family>
     </font>
    </property>
    <property name="text">
     <string>Max Generaciones</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="numGeneraciones">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>350</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Sans Serif</family>
     </font>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
